### PR TITLE
fix: fixed all the broken links and formatted the file structure

### LINKED ISSUE #240: [BUG]: Fixing all the broken links in the footer section of Student's main page and Stakeholder's main page
**Describe the bug**
The footer section of Student's main page and Stakeholder's main page contain a lot of broken links.

**To Reproduce**
1. Login as Student and got to the footer of the page.
2. Do the same for Stakeholder.

**Expected behavior**
The links should be redirecting to appropriate pages/sections (e.g. Privacy policy, about us)

**Additional context**
None

@Harshdev098 Please assign this issue to me and add 'hacktoberfest', 'gssoc-ext' and 'level' labels to it.

### PR DESCRIPTION
**Pull request by trinetra110**
## 📋 Description

Fixed all the broken links and formatted the file structure of some files for proper understanding.

## 🔨 Changes Made

- Fixed all the broken links in 7 files
- Formatted the file structure in 7 files

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [x] I have followed all the guidelines mentioned in CONTRIBUTING.md.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [x] I have tested it locally and it works fine.
- [x] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [x] Bug fix (non-breaking change which fixes an issue) 🐛
- [ ] New feature (non-breaking change which adds functionality) ✨
- [ ] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

- Fixes #240 

## 📸 Screenshots (if applicable)

---

*Thank you for contributing to Research Nexus! We look forward to reviewing your PR and getting it merged!*

### CODE REVIEW

**Comment by trinetra110** [comment]
@Harshdev098 I have made the required changes. Please review it and add the 'hacktoberfest', 'hacktoberfest-accepted', 'gssoc-ext' and 'level3' tags.

File: public/main_page.html [hunk 1]
```diff
@@ -89,14 +89,14 @@ <h2 style="padding: 10px;font-size: 44px;color: rgb(202, 60, 60);">Explore Resea
                     <button type="submit">Search</button>
                 </form>
                 <ul>
-                    <li><a href="">Home</a></li>
-                    <li><a href="">About</a></li>
-                    <li><a href="">Add Topic</a></li>
-                    <li onmouseover="show()" onmouseout="hide()"><a href="" title="View Notifications"><i id="firsticon"
-                                class="fa-sharp fa-regular fa-bell"
+                    <li><a href="/">Home</a></li>
+                    <li><a href="/#about_show">About</a></li>
```
Comment (Harshdev098) [inline public/main_page.html:93]: remove this list item

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

File: public/main_page.html (hunk 1 above)
Comment (trinetra110) [reply public/main_page.html:93]: @Harshdev098 Which list item? Home, about, notification, or all of them?

Comment (Harshdev098) [reply public/main_page.html:93]: only about

Comment (trinetra110) [reply public/main_page.html]: Ok done

**Comment by trinetra110** [comment]
@Harshdev098 Conflict resolved and changes done. Please check.

**Comment by Harshdev098** [comment]
there are conflicts resolve them, please!

**Review by Harshdev098 — APPROVED** [review]

**Pull request merged**